src/main/java/com/synopsys/integration/alert/web/model/FieldValueModel.java

```diff
@@ -48,8 +53,8 @@ public void setValues(final Collection<String> values) {
         return values.stream().findFirst();
     }
 
-    public void setValue(String value) {
-        this.values = Arrays.asList(value);
+    public void setValue(final String value) {
+        values = Arrays.asList(value);
```

Comment:
Let's use Set.of(value) for the singleton case. I know you didn't add the Arrays.asList(), but as these cases come to light, we should convert them to the Java 11 standard (let's also use Set.of() for the constructor.